netbout-web/src/main/java/com/netbout/dynamo/DyAttachment.java

```diff
@@ -188,40 +187,7 @@ public InputStream read() throws IOException {
     @Override
     public void write(final InputStream stream, final String ctype,
         final String etag) throws IOException {
-        if (etag.isEmpty()) {
```

Comment:
@exper0 why did you remove `etag` and `ctag` validation ?
I hadn't saw these validations in TkAttach